app/models/green_lanes/category_assessment.rb

```diff
@@ -50,7 +50,7 @@ def load_from_file(file = DEFAULT_JSON)
 
       def load_from_string(data)
         json_array = JSON.parse(data)
-        @all = json_array.map { |json| new(json) }
+        @all = json_array.map { |json| new(json).tap(&:id).tap(&:freeze) }
```

Comment:
so, to check, this is returning the same object as before,
but it freezes the IDs so they cannot be changed?

why do we need it?

Reply:
The ContentAddressableId module stores the id in `@id` after generating it -> can't do this on a frozen class so we force generation prior to freezing